openpifpaf/plugins/apollocar3d/.gitignore

```diff
@@ -0,0 +1,55 @@
+# system
+.DS_Store
+.vscode
+*.so
+.idea/
+
+# Python
+build/
+venv*/
+*.egg-info
+.eggs/
+__pycache__
+dist/
+.pytest_cache/
+.ipynb_checkpoints/
+openpifpaf/functional.cpython-*
+openpifpaf/functional.html
+
+# guide
+guide/_build/
+guide/coco/*.predictions.png
+guide/coco/*.predictions.json
+guide/logo.png
+guide/favicon.png
+guide/profile_decoder.1.json
+guide/profile_decoder.prof
+guide/profile_decoder_flame.svg
+
+# data
+all-images/
+data/
+data-*/
+data_*/
+docs-private/
+docs/*/*.predictions.png
+data-mscoco
+*debug.png
+*.pl
+*.pl-*
+*.pl.*
+*.pkl
+*.pkl-*
+*.pkl.*
+*.log
+*.zip
+outputs
+outputs/
+shared/
+decoder.prof
+decoder_flame.svg
+*.onnx
+debug.*.png
+test-clis-*/
+*.mlmodel
+slurm*
```

Comment:
Maybe move this `slurm*` to the main `.gitignore` and remove the file. Multiple `.gitignore` files might be confusing.